src/jit/importer.cpp

```diff
@@ -4709,6 +5018,14 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
                                 }
                             }
                         }
+                        else if (strcmp(methodName, "GetElement") == 0)
```

Comment:
nit: Might be nice to keep these alphabetical, so it is easier to see what is and isn't covered.

Reply:
Will do.